ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml

```diff
@@ -0,0 +1,67 @@
+workflow:
+  as: hypershift-mce-kubevirt-baremetalds-disconnected-conformance
+  documentation: |-
+    The HyperShift KubeVirt conformance workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+    This is executed in a fully disconencted cluster with
+    mirrored images.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in Azure account and is destroyed after tests complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team.
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - ref: hypershift-mce-dump
+    - chain: gather-core-dump
+    - chain: hypershift-mce-kubevirt-destroy
+    - chain: baremetalds-ipi-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - chain: baremetalds-ipi-pre
+    - ref: hypershift-mce-install-disconnected
```

Comment:
this step is pinned to MCE 2.4.  that will only be accurate for how we're testing ocp 4.14.

today we have the following periodics
ocp 4.14 + mce 2.4 + cnv 4.14
ocp 4.15 + mce 2.5 + cnv 4.15
ocp 4.16 + latest hypershift-operator (switches to mce 2.6 pre-release soon) + cnv 4.16

Reply:
it takes the value from:
      MCE_VERSION: "2.4"
on the definition of the periodic job: other jobs will bring different values.